lib/Settings.js

```diff
@@ -3,6 +3,8 @@
 var Cesium = require('cesium');
 var nconf = require('nconf');
 var Promise = require('bluebird');
+var fs = require('fs');
```

Comment:
`fs-extra` should probably be used instead (since it has direct promise support and we alreayd depend on it in the code), though you probably want to promote it to `dependency` instead of `devDependency`